app/controllers/staff/events_controller.rb

```diff
@@ -63,6 +63,15 @@ def update
     end
   end
 
+  def open_cfp
+    if @event.update_attributes(state: "open", opens_at: DateTime.now) && (@event.status == Event::STATUSES[:open])
+      flash[:info] = "Your CFP was successfully opened."
+    else
+      flash[:danger] = 'There was a problem opening your CFP. Please check your closing date and try again.'
```

Comment:
I would drop off the 'Please check..' part since it's just guessing what the issue is and add the @event errors.full_messages.to_sentence to the end instead.  That assumes you the open_cfp method on event adds error messages as to why it failed to open.